### PR TITLE
HPCC-28510 Pass sort param into LogicalFile detail tabs

### PR DESCRIPTION
**Pull request by jeclrsg**
Signed-off-by: Jeremy Clements <Jeremy.Clements@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src-react/components/FileDetailsGraph.tsx [hunk 1]
```diff
@@ -63,7 +69,7 @@ export const FileDetailsGraph: React.FunctionComponent<FileDetailsGraphProps> =
     const buttons = React.useMemo((): ICommandBarItemProps[] => [
         {
             key: "refresh", text: nlsHPCC.Refresh, iconProps: { iconName: "Refresh" },
-            onClick: () => refreshData()
+            onClick: () => refreshTable()
```
Comment (GordonSmith) [inline esp/src/src-react/components/FileDetailsGraph.tsx:72]: Can you double check that this is fetching data from the server?

File: esp/src/src-react/components/FileHistory.tsx [hunk 2]
```diff
@@ -56,14 +56,14 @@ export const FileHistory: React.FunctionComponent<FileHistoryProps> = ({
     const buttons: ICommandBarItemProps[] = React.useMemo(() => [
         {
             key: "refresh", text: nlsHPCC.Refresh, iconProps: { iconName: "Refresh" },
-            onClick: () => refreshData()
+            onClick: () => refreshTable()
```
Comment (GordonSmith) [inline esp/src/src-react/components/FileHistory.tsx:59]: Can you double check that this is fetching data from the server?

File: esp/src/src-react/components/FileParts.tsx [hunk 3]
```diff
@@ -60,9 +70,9 @@ export const FileParts: React.FunctionComponent<FilePartsProps> = ({
     const buttons = React.useMemo((): ICommandBarItemProps[] => [
         {
             key: "refresh", text: nlsHPCC.Refresh, iconProps: { iconName: "Refresh" },
-            onClick: () => refreshData()
+            onClick: () => refreshTable()
```
Comment (GordonSmith) [inline esp/src/src-react/components/FileParts.tsx:73]: Can you double check that this is fetching data from the server?

File: esp/src/src-react/components/ProtectedBy.tsx [hunk 4]
```diff
@@ -52,9 +52,9 @@ export const ProtectedBy: React.FunctionComponent<ProtectedByProps> = ({
     const buttons = React.useMemo((): ICommandBarItemProps[] => [
         {
             key: "refresh", text: nlsHPCC.Refresh, iconProps: { iconName: "Refresh" },
-            onClick: () => refreshData()
+            onClick: () => refreshTable()
```
Comment (GordonSmith) [inline esp/src/src-react/components/ProtectedBy.tsx:55]: Can you double check that this is fetching data from the server?

File: esp/src/src-react/components/SuperFiles.tsx [hunk 5]
```diff
@@ -53,7 +53,7 @@ export const SuperFiles: React.FunctionComponent<SuperFilesProps> = ({
     const buttons = React.useMemo((): ICommandBarItemProps[] => [
         {
             key: "refresh", text: nlsHPCC.Refresh, iconProps: { iconName: "Refresh" },
-            onClick: () => refreshData()
+            onClick: () => refreshTable()
```
Comment (GordonSmith) [inline esp/src/src-react/components/SuperFiles.tsx:56]: Can you double check that this is fetching data from the server?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Review by GordonSmith — APPROVED** [review]

**Review by ghalliday — APPROVED** [review]

**Pull request merged**